contracts/protocol/integration/ZeroExApiAdapter.sol

```diff
@@ -0,0 +1,136 @@
+/*
+    Copyright 2021 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    SPDX-License-Identifier: Apache License, Version 2.0
+*/
+
+pragma solidity 0.6.10;
+pragma experimental "ABIEncoderV2";
+
+/**
+ * @title ZeroExApiAdapter
+ * @author Set Protocol
+ *
+ * Exchange adapter for 0xAPI that returns data for swaps
+ */
+
+contract ZeroExApiAdapter {
+
+    /* ============ State Variables ============ */
+
+    // Address of the deployed ZeroEx contract.
+    address public immutable zeroExAddress;
+
+    // Returns the address to approve source tokens to for trading. This is the TokenTaker address
+    address public immutable getSpender;
+
+    /* ============ constructor ============ */
+
+    constructor(address _zeroExAddress) public {
+        zeroExAddress = _zeroExAddress;
+        getSpender = _zeroExAddress;
```

Comment:
Want to double-check this. The spender is the 0x address. The comment above refers to a `TokenTaker` wanna make sure that's the same address.